crates/matrix-sdk/src/store.rs

```diff
@@ -33,3 +33,55 @@
 pub use matrix_sdk_indexeddb::*;
 #[cfg(any(feature = "sled_state_store", feature = "sled_cryptostore"))]
 pub use matrix_sdk_sled::*;
+
```

Comment:
Yeah, this doesn't make me happy at all. I understand the reasons behind this are because of the features-mess ... but if we must move this back in here, can we at least put a large FIXME and issue up to resolve this?  

I do understand correctly, that the same problem this would attempt to solve, is also present with the indexeddb-store actually, right?

Reply:
Well yes and no. The likelihood that anyone will use the indexeddb crypto store is much lower so it's less important to support this over there. For consistency sake, we probably should do the same for the indexeddb store.